bridges/ethElrond/bridgeExecutor.go

```diff
@@ -147,7 +147,11 @@ func (executor *bridgeExecutor) MyTurnAsLeader() bool {
 
 // GetBatchFromElrond fetches the pending batch from Elrond
 func (executor *bridgeExecutor) GetBatchFromElrond(ctx context.Context) (*clients.TransferBatch, error) {
-	return executor.elrondClient.GetPending(ctx)
+	batch, err := executor.elrondClient.GetPending(ctx)
+	if err == nil {
+		executor.statusHandler.SetIntMetric(core.MetricNumBatches, int(batch.ID)-1)
```

Comment:
can you clean the core/constants.go of unused definitions?

Reply:
I will do another PR for cleaning and also adding extra metrics.